libpod/container_api.go

```diff
@@ -705,3 +705,23 @@ func (c *Container) Sync() error {
 
 	return nil
 }
+
+// Restart restarts a running container and takes a given timeout in uint
```

Comment:
Lint probably going to complain this isn't `// RestartWithTimeout restarts...`

Reply:
nice catch!